### PR TITLE
Customized indent for the generated validating builders code

### PR DESCRIPTION
**Pull request by IlliaShepilov**
Summary of changes:
- Migrated to 0.8.13-SNAPSHOT.
- Added the possibility to customize indent in the generated code.

To configure the `indent` for the generated code need to add the next snippet to the `build.gradle`:
```gradle
modelCompiler {
    indent = 4
}
```

### CODE REVIEW

**Comment by IlliaShepilov** [comment]
@armiol PTAL.

**Comment by IlliaShepilov** [comment]
@armiol PTAL.

File: spine-model-compiler/plugin/src/main/java/org/spine3/gradle/compiler/Indent.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    private final int indent;
+
+    public Indent(int indent) {
+        this.indent = indent;
+    }
+
+    public String getJavaPoetIndent() {
+        final StringBuilder builder = new StringBuilder(indent);
+        for (int i = 0; i < indent; i++) {
+            builder.append(' ');
```
Comment (armiol) [inline spine-model-compiler/plugin/src/main/java/org/spine3/gradle/compiler/Indent.java]: Please have a look at [`Strings.repeat`](https://google.github.io/guava/releases/19.0/api/docs/com/google/common/base/Strings.html#repeat(java.lang.String, int)).

File: spine-model-compiler/plugin/src/main/java/org/spine3/gradle/compiler/validate/ValidatorsGenPlugin.java [hunk 2]
```diff
@@ -55,8 +57,17 @@
...
+ *      }
  * }
+ *
+ * <p>To configure indent for the generated code need to add
+ * the next snippet to the build.gradle file:
+ *
+ * {@code
+ * modelCompiler{
+ *    indent = 4
+ *      }
```
Comment (armiol) [inline spine-model-compiler/plugin/src/main/java/org/spine3/gradle/compiler/validate/ValidatorsGenPlugin.java]: Let's fix the formatting and also use `<pre>`. Same applies to the PR description.

File: spine-model-compiler/plugin/src/main/java/org/spine3/gradle/compiler/validate/ValidatorsGenPlugin.java [hunk 3]
```diff
@@ -55,8 +57,17 @@
  *
  * {@code
  * modelCompiler{
- * generateValidators = false
+ *    generateValidatingBuilders = false
+ *      }
  * }
+ *
+ * <p>To configure indent for the generated code need to add
```
Comment (armiol) [inline spine-model-compiler/plugin/src/main/java/org/spine3/gradle/compiler/validate/ValidatorsGenPlugin.java]:
```
<p>The tabular indentation for the generated code is done with whitespaces. To set the width please use the `indent` property as follows:

<your sample>

<p>The default value is 4.
```

Please update the description of `generateValidatingBuilders` parameter accordingly.

**Comment by IlliaShepilov** [comment]
@armiol PTAL.

File: spine-model-compiler/plugin/src/main/java/org/spine3/gradle/compiler/Indent.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.spine3.gradle.compiler;
+
+import com.google.common.base.Strings;
+
+/**
+ * A non-negative indent of generated code.
```
Comment (armiol) [inline spine-model-compiler/plugin/src/main/java/org/spine3/gradle/compiler/Indent.java]: It's not `non-negative`. It's positive, as zero is not allowed.

File: spine-model-compiler/plugin/src/main/java/org/spine3/gradle/compiler/validate/ValidatorsGenPlugin.java [hunk 5]
```diff
@@ -50,15 +52,33 @@
  *
  * <p>Logs a warning if there are no Protobuf descriptors generated.
  *
- * <p>To switch off the generation of the validating builders need
- * to add the next snippet to the build.gradle file:
+ * <p>The validating builders are generated by default.
```
Comment (armiol) [inline spine-model-compiler/plugin/src/main/java/org/spine3/gradle/compiler/validate/ValidatorsGenPlugin.java]: Please remove this line, but keep `<p>`. The reason is that you are already mentioning its default value below.

**Pull request merged**